tests/e2e/pageObjectModel/index.js

```diff
@@ -0,0 +1,85 @@
+import path from "path";
+import { expect, test } from "@playwright/test";
+import { findLatestBuild, parseElectronApp } from "electron-playwright-helpers";
+import { _electron as electron } from "playwright";
+
+class ModVApp {
+  /**
+   * @param {import('@playwright/test').Page} page
+   */
+  constructor() {
+    this.electronApp;
+
+    test.beforeAll(async () => {
+      // find the latest build in the out directory
+      const latestBuild = findLatestBuild(path.resolve("./dist_electron"));
+      // parse the directory and find paths and other info
+      const appInfo = parseElectronApp(latestBuild);
+      // set the CI environment variable to true
+      process.env.CI = "e2e";
+
+      this.electronApp = await electron.launch({
+        args: [appInfo.main],
+        executablePath: appInfo.executable
+      });
+
+      this.electronApp.on("window", async page => {
+        // const filename = page
+        //   .url()
+        //   ?.split("/")
+        //   .pop();
+        // console.log(`Window opened: ${filename}`);
+
+        // capture errors
+        page.on("pageerror", error => {
+          console.error(error);
+        });
+        // capture console messages
+        page.on("console", msg => {
+          console.log(msg.text());
+        });
+      });
+    });
+
+    test.afterAll(() => {
+      // Playwright, why are you so bad at timings?
```

Comment:
Unhappy about this hack, but it seems needed.

Luckily it seems like it's only teardown timing, so doesn't affect the running of the tests until the last.